qemu/tests/qemu_guest_agent.py

```diff
@@ -701,13 +701,19 @@ def gagent_check_set_vcpus(self, test, params, env):
         error_context.context("the vcpu number:%d" % vcpus_num, LOG_JOB.info)
         if vcpus_num < 2:
             test.error("the vpus number of guest should be more than 1")
-        vcpus_info[vcpus_num - 1]["online"] = False
-        del vcpus_info[vcpus_num - 1]["can-offline"]
-        action = {'vcpus': [vcpus_info[vcpus_num - 1]]}
-        self.gagent.set_vcpus(action)
+        while True:
+            random_index = random.randint(1, vcpus_num)
```

Comment:
Hi @6-dehan 

The range should be (0, vcpus_num - 1), could you please help check?
Thanks!

Reply:
yeah, you're right. updated it. thanks